worldedit-core/src/main/java/com/sk89q/worldedit/world/block/BaseBlock.java

```diff
@@ -192,8 +192,18 @@ public int hashCode() {
         return ret;
     }
 
+    /**
+     * Gets a string representation of this BaseBlock.
```

Comment:
Can we be more specific about the format? This method isn't useful without strict details on the output.

Reply:
Can I be specific to the extent that we state it complies with MC's formatting / the parser's formatting? i'd rather not lock us into formatting by API contract